radsat-sk/operation/crypt/RCrypt.c

```diff
@@ -0,0 +1,66 @@
+#include "RCrypt.h"
+
+int getPaddedBufSize(int plaintextLen);
+void padBuffer(uint8_t* buff, uint8_t* paddedBuff, int buffLen, int newBuffLen);
+void encrypt(uint8_t* plaintext, int plaintextLen);
+size_t decrypt(uint8_t* ciphertext, int ciphertextlen);
+
+
+
+/**
+ * @brief Gets the next multiple of 16 from the size of the input buffer
+ * @header "operations/crypt/encrypt.h"
+ * @param plaintextLen - The size of the unpadded buffer containing data to be encrypted.
+ * @pre need to have a variable storing the size of the plaintext buffer
+ * @return The size of the new buffer to be created
+ */
+int getPaddedBufSize(int plaintextLen){
```

Comment:
1. Don't use abreviations (Len should be Length)
2. Don't non-standard types, such as int. Use a standard type, like `uint16_t`
3. Check your coding style; need a space before the curly bois